index.js

```diff
@@ -76,7 +76,7 @@ require("./middleware/authEmail")(passport);
 require("./middleware/authGithub")(passport);
 
 // index route
-app.use("/api", indexRouter);
+app.use("/api/auth/api", indexRouter);
```

Comment:
The indexRouter holds other endpoints like `/api/authorize` for instance. Doing this changes all that endpoint to `/api/auth/api/authorize`.

why?

Is this so that we can host all api's on the same server?

Reply:
This is a test for the deployment on nginx server, yes, all endpoints will follow this if it works on the new server.